lib/bookmark_reminder_notification_handler.rb

```diff
@@ -30,7 +33,7 @@ def self.create_notification(bookmark)
     user = bookmark.user
     user.notifications.create!(
       notification_type: Notification.types[:bookmark_reminder],
-      topic_id: bookmark.topic_id,
+      topic_id: bookmark.topic.id,
```

Comment:
technically we can just call `topic_id` since we set up the delegation in the bookmark model.